apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -72,3 +75,19 @@ div.pl-order-blocks-container {
   display: flex;
   align-items: center;
 }
+
+.pl-order-blocks-pairing-indicator {
+  border-radius: 10px;
+  border: 0px !important;
+  list-style: none;
+  margin-left: -10px;
+  margin-right: -10px;
+  padding: 4px 10px 4px 10px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  overflow: hidden;
+}
```

Comment:
I'll overlook the pixel values since the rest of the element already uses them, but we typically want to use `rem` values instead.

Reply:
noted